msi/GitProduct.wxs

```diff
@@ -58,11 +62,25 @@
     <SetProperty Id="ENABLEFSCACHE" Value="[CMDLINE_ENABLEFSCACHE]" After="AppSearch" Sequence="first">CMDLINE_ENABLEFSCACHE</SetProperty>
 
     <!-- ENABLEGCM Property implementing the "Remember Property" pattern -->
-    <Property Id="ENABLEGCM" Value="1">
+    <!-- Initially set this to disabled until we can determine that .NET Framework 4.5.1 and above is installed. -->
+    <Property Id="ENABLEGCM" Value="0" />
```

Comment:
An easier way to determine whether the user set this property would be to initialize it to `"-1"`...